src/models/gpayCovidCard/index.ts

```diff
@@ -5,14 +5,19 @@ import GPay, {
 } from "@notarise-gov-sg/gpay-covid-cards";
 import { Bundle } from "../../models/fhir/types";
 import { isoToDateOnlyString, isoToLocaleString } from "../../common/datetime";
+import { getDefaultIfUndefined } from "../../config";
 
 const genGPayCovidCardUrl = (
-  gpaySigner: { issuer: string; issuerId: string; privateKey: string },
+  gpaySigner: { issuer: string; issuerId: string },
   parsedFhirBundle: Bundle,
   uuid: string,
   storedUrl: string
 ) => {
-  const gpay = GPay(gpaySigner.privateKey);
+  const privateKey = getDefaultIfUndefined(
+    process.env.GPAY_COVID_CARD_PRIVATE_KEY,
+    ""
+  ).replace(/\\n/g, "\n");
```

Comment:
Are we able to move this to `config.ts`? So that we can still use `gpaySigner.privateKey`?

Reply:
I tried that before, but it doesn't work. if i put in config.ts, the env value was empty. the middleware only pass the env value to function handler and imported files. Unable to pass other than that, it's weird